### DISCUSSION TITLE
Anaconda is not free for commercial use

### DISCUSSION THREAD

**Discussion by edvardoss** [IyarLin/R-advantages-over-python#10] [General]
I read this [article ](https://www.reddit.com/r/Python/comments/iqsk3y/anaconda_is_not_free_for_commercial_use_anymore/) and thought that now one of the advantages of R in front of Python.
But then I read this [article ](https://www.r-bloggers.com/2019/01/how-gpl-makes-me-leave-r-for-python/) and so began to doubt. Although in the last link, it seems to be not talking about the commercial use of the language itself, but about the licensing protection of products developed on it, so it seems that now in the context of the news about Anaconda - R have more advantages.

**Comment by IyarLin** [comment]
Thanks for bringing this up, I wasn't aware. I'm having difficulties convincing the CTO where I work to use R a lot due to that license issue. Hope it get's resolved some time.
